fbw-a32nx/src/systems/instruments/src/util.js

```diff
@@ -1,107 +1,111 @@
+// Copyright (c) 2021-2023 FlyByWire Simulations
+//
+// SPDX-License-Identifier: GPL-3.0
+
 import { useEffect, useState, useRef } from 'react';
 
 export const renderTarget = document.getElementById('MSFS_REACT_MOUNT');
 export const customElement = renderTarget.parentElement;
 
 // @param {() => void} handler
 export function useInteractionEvent(event, handler) {
-    // Logic based on https://usehooks.com/useEventListener/
-    const savedHandler = useRef(handler);
-    useEffect(() => {
-        savedHandler.current = handler;
-    }, [handler]);
-
-    useEffect(() => {
-        const wrappedHandler = (e) => {
-            if (event === '*') {
-                savedHandler.current(e.detail);
-            } else {
-                savedHandler.current();
-            }
-        };
-        customElement.addEventListener(event, wrappedHandler);
-        return () => {
-            customElement.removeEventListener(event, wrappedHandler);
-        };
-    }, [event]);
+  // Logic based on https://usehooks.com/useEventListener/
+  const savedHandler = useRef(handler);
+  useEffect(() => {
+    savedHandler.current = handler;
+  }, [handler]);
+
+  useEffect(() => {
+    const wrappedHandler = (e) => {
+      if (event === '*') {
+        savedHandler.current(e.detail);
+      } else {
+        savedHandler.current();
+      }
+    };
+    customElement.addEventListener(event, wrappedHandler);
+    return () => {
+      customElement.removeEventListener(event, wrappedHandler);
+    };
+  }, [event]);
 }
 
 // @param {(deltaTime: number) => void} handler
 export function useUpdate(handler) {
-    // Logic based on https://usehooks.com/useEventListener/
-    const savedHandler = useRef(handler);
-    useEffect(() => {
-        savedHandler.current = handler;
-    }, [handler]);
-
-    useEffect(() => {
-        const wrappedHandler = (event) => {
-            savedHandler.current(event.detail);
-        };
-        customElement.addEventListener('update', wrappedHandler);
-        return () => {
-            customElement.removeEventListener('update', wrappedHandler);
-        };
-    });
+  // Logic based on https://usehooks.com/useEventListener/
+  const savedHandler = useRef(handler);
+  useEffect(() => {
+    savedHandler.current = handler;
+  }, [handler]);
+
+  useEffect(() => {
+    const wrappedHandler = (event) => {
+      savedHandler.current(event.detail);
+    };
+    customElement.addEventListener('update', wrappedHandler);
+    return () => {
+      customElement.removeEventListener('update', wrappedHandler);
+    };
+  });
 }
 
 const SIMVAR_TYPES = {
-    '__proto__': null,
-    'GPS POSITION LAT': 'degrees latitude',
-    'L:A32NX_COLD_AND_DARK_SPAWN': 'Bool',
-    'TOTAL AIR TEMPERATURE': 'Celsius',
-    'AMBIENT TEMPERATURE': 'Celsius',
+  __proto__: null,
```

Comment:
Is this intended to be unquoted? Concern regarding how this is handled, especially if the intention is to not set the object prototype